cypress.config.js

```diff
@@ -1,9 +1,25 @@
 const { defineConfig } = require('cypress');
+const { faker } = require('@faker-js/faker');
 
 module.exports = defineConfig({
   e2e: {
+    baseUrl: 'https://demoqa.com/',
+
     setupNodeEvents(on, config) {
-      // implement node event listeners here
-    }
-  }
+      on('task', {
+        generateUser() {
+          randomIndex = Math.floor(Math.random()*100);
+          return{
+            firstName: faker.person.firstName() + randomIndex,
+            lastName: faker.person.lastName() + randomIndex,
+            email: randomIndex + faker.internet.email(),
+            age: faker.date.birthdate('age'),
```

Comment:
```suggestion
            age: faker.date.birthdate('age'),
```
this generate code in format '1968-01-06T22:07:36.218Z'

Reply:
this generate code in format '1968-01-06T22:07:36.218Z' if the age: faker.date.birthdate(), - with no arguments in parentheses. But if we would put the 'age' into the parentheses like age: faker.date.birthdate('age'), so we would have valid age. I have read it in the faker documentation before using it. Here are screenshots of my code and cypress
![Age1](https://github.com/mate-academy/qa_cypress_e2e_web_tables/assets/158562893/36b2e4a0-7b85-454f-badd-66fc82082da9)
![Age2](https://github.com/mate-academy/qa_cypress_e2e_web_tables/assets/158562893/0017130a-12d1-46cb-b6a7-e2c363fb31d4)

Reply:
<img width="1387" alt="image" src="https://github.com/mate-academy/qa_cypress_e2e_web_tables/assets/85580286/c6c214ab-87de-4124-aca6-6dda0575b23a">
you type this value

Reply:
Yes, but it is shown as 19 in the form. That's all I need)